docs/user/vulnerabilities/vulnerability/product_status/under_investigation-usage.en.md

```diff
@@ -1,14 +1,22 @@
 # Under Investigation - Usage
 
-* A corresponding Full Product Name element with matching `product_id` must be defined.
+For the products listed here, it is not yet clear whether they are affected or not affected.
+It is expected that in a later release the analysis result will be communicated.
 
-* A product listed here must not be listed in any of the following
+If general mitigation guidance can be provided and is applicable to products currently under investigation, it should be provided.
+A valid remediation can also be of the category `none_available` or `no_fix_planned`.
 
-  * `product_status/first_affected[]`
-  * `product_status/known_affected[]`
-  * `product_status/last_affected[]`
-  * `product_status/known_not_affected[]`
-  * `product_status/first_fixed[]`
-  * `product_status/fixed[]`
+Especially useful in cases of large Multi-Party Coordinated Vulnerability Disclosure to provide the current status of the investigation.
 
-* For each product ID listed here, a remediation must exist.
+A product listed here must not be listed in any of the following
+
+* `product_status/first_affected[]`
+* `product_status/known_affected[]`
+* `product_status/last_affected[]`
+* `product_status/known_not_affected[]`
+* `product_status/first_fixed[]`
+* `product_status/fixed[]`
+
+For each product ID listed here, a remediation should exist.
```

Comment:
I guess that doesn't make sense as we don't know yet whether they are affected...

Reply:
removed